client/helpers.go

```diff
@@ -192,9 +192,22 @@ const (
 	RedshiftService             AWSService = "redshift"
 	Route53Service              AWSService = "route53"
 	S3Service                   AWSService = "s3"
+	WAFRegional                 AWSService = "waf-regional"
 	WorkspacesService           AWSService = "workspaces"
 )
 
+// MakeARN creates an ARN using supplied service name, account id, region name and resource id parts.
+// Resource id parts are concatenated using forward slash (/).
+func MakeARN(service AWSService, accountID, region string, idParts ...string) string {
```

Comment:
Maybe use this inside `helpers.resolveARN()` as well?

Reply:
It is used there.. :)